### PR TITLE
General review PR

### PR DESCRIPTION
**Pull request by prasanna1211**

### CODE REVIEW

File: src/autocomplete/readMe.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## AutoComplete
+
+A basic autocomplete component.
+
+### Properties
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| `data` | `Array` | `Required` | An array of options for autocompletion |
```
Comment (prasanna1211) [inline src/autocomplete/readMe.md:8]: Default is required ? I feel default option should be N/A and required should be highlighted elsewhere.

File: src/avatar/readMe.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Avatar
+
+The base avatar component.
```
Comment (prasanna1211) [inline src/avatar/readMe.md:3]: Base Avatar ?

Comment (prasanna1211) [reply src/avatar/readMe.md:3]: Similarly change at card docs too.

File: src/checkbox/readMe.md [hunk 3]
```diff
@@ -1,63 +1,51 @@
...
-| `size` | `String` | `medium`  | Defines the size of the button. Possible values are `small`, `medium`, and `large` |
-| `className` | `String` | `''` | Set a class to style the Component |
-| `href` | `String` | `null` | Creates a link for the button. |
-| `icon` | `String`/`Element` | `null` | Value of the icon |
-| `iconAlignment` | `String` | `left` | Set an alignment of the icon. Possible values are `left`, `right` |
-| `onClick` | `Function` | &nbsp; | Callback to be called when the button is clicked |
+| `checked` | `boolean` | `false` | Prop that sends in the state of toggle |
+| `onClick` | `Function` | &nbsp; | Callback to be called when checked or unchecked |
+| `additionalClass` | `String` | &nbsp; | Prop to send additional classNames for styling |
+| `label` | `String` | `check` | Prop to send when a callback value is required onClick |
```
Comment (prasanna1211) [inline src/checkbox/readMe.md:11]: Is this description correct for the label ?

File: src/autocomplete/readMe.md (hunk 1 above)
Comment (KRRISH96) [reply src/autocomplete/readMe.md:8]: Fixed in #57

File: src/avatar/readMe.md (hunk 2 above)
Comment (KRRISH96) [reply src/avatar/readMe.md:3]: Fixed in #57

File: src/checkbox/readMe.md (hunk 3 above)
Comment (KRRISH96) [reply src/checkbox/readMe.md:11]: Corrected in #57

**Pull request closed without merging**